### ISSUE TITLE
Setup with Caddy

### ISSUE THREAD

**Issue by mitexleo** [matrix-org/sliding-sync#396]
Hi,
I'm running matrix at matrix.nixorigin.one and I'm using Caddy as a reverse proxy. I have some some questions about the setup of Sliding sync:

1. Do I need the .well-known config even though I'm running matrix at a subdomain ?
2. Do I need to use a separate database for Syncv3?

Thanks

**Comment by axelsimon** [comment]
Yes, you need to declare in your .well-known where to find the `org.matrix.msc3575.proxy` so that compatible clients can know where to find it.

What do you mean by separate? It can be the same Postgres instance, but it needs to be a different database on that instance (ie: not the `synapse` db used by `synapse_user`, if you followed the instructions from Synapse). That is why there is a `createdb` command suggested in the setup instructions.
